packages/admin/src/routes/dashboard/components/Dashboard/Dashboard.js

```diff
@@ -1,23 +1,55 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import {Typography, Icon} from 'tocco-ui'
+import {Typography, theme} from 'tocco-ui'
 import styled from 'styled-components'
 
+import ToccoLogo from '../../../../assets/tocco_red.svg'
+
+const StyledDashBoardWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  background-color: ${theme.color('paper')}
+`
+
+const InnerWrapper = styled.div`
+  width: 600px;
+  height: 50px;
+  position: relative;
+  left: -320px
+`
+
 const StyledDashBoard = styled.div`
   display: flex;
-  justify-content: center;
+  height: 75%;
+  justify-content: flex-end;
+  align-items: flex-end;
+  margin-right: -180px;
   
-  div { 
-      padding:  15px;
+  && {
+    * {
```

Comment:
Are you using the asterisk selector to select the tile and later to select the version text? Isn't there a more transparent way to to that?